wfe/web-front-end.go

```diff
@@ -268,7 +268,7 @@ func (wfe *WebFrontEndImpl) NewRegistration(response http.ResponseWriter, reques
 		return
 	}
 
-	regURL := fmt.Sprintf("%s%d", wfe.RegBase, reg.ID)
+	regURL := fmt.Sprintf("%s%s", wfe.RegBase, string(reg.ID))
```

Comment:
I'm pretty sure this is wrong. I changed this in a previous pull request: https://github.com/letsencrypt/boulder/pull/208#discussion-diff-30633294. The ID was getting output as a single Unicode character rather than a number.

The current code does generate a `go get` complaint, which I fixed (in a hacky way) in this PR: https://github.com/letsencrypt/boulder/pull/215